test/unit/plugins.test.js

```diff
@@ -1,206 +1,210 @@
+/* eslint-env qunit */
 import {IE_VERSION} from '../../src/js/utils/browser';
 import registerPlugin from '../../src/js/plugins.js';
 import Player from '../../src/js/player.js';
 import TestHelpers from './test-helpers.js';
 import window from 'global/window';
+import sinon from 'sinon';
```

Comment:
if we're importing sinon directly, we should remove it from the `global-shim.js` file.